### PR TITLE
improvement(kms): add check for KMS encryption fact

### PR DESCRIPTION
**Pull request by vponomaryov**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by vponomaryov** [comment]
Blocked by this: https://github.com/scylladb/scylla-enterprise/issues/3461

File: sdcm/utils/sstable/sstable_utils.py [hunk 1]
```diff
@@ -53,6 +51,32 @@ def get_sstables(self, from_minutes_ago: int = 0):
...
+        if not sstables:
+            sstables = self.get_sstables()
+        if isinstance(sstables, str):
+            sstables = [sstables]
+        sstables_encrypted_mapping = {}
+        for sstable in sstables:
+            sstables_res = self.db_node.remoter.sudo(
+                f"{self.db_node.add_install_prefix('/usr/bin/scylla')} sstable dump-scylla-metadata"
+                # NOTE: uncomment below when https://github.com/scylladb/scylla-enterprise/issues/3461 gets fixed
+                # "  --scylla-yaml-file /etc/scylla/scylla.yaml"
```
Comment (vponomaryov) [inline sdcm/utils/sstable/sstable_utils.py:64]: Looks like it is required for newly created tables: https://github.com/scylladb/scylla-enterprise/issues/3461#issuecomment-1740881817
And https://github.com/scylladb/scylla-enterprise/issues/3461 comes out to be the blocker

Comment (vponomaryov) [reply sdcm/utils/sstable/sstable_utils.py]: False. It was problem with the scylla-bench commands.

**Comment by vponomaryov** [comment]
Tested here:
- 3 nodes, RF=3: https://jenkins.scylladb.com/job/scylla-staging/job/valerii/job/vp-longevity-10gb-3h-test/55
- 6 nodes, RF=3: https://jenkins.scylladb.com/job/scylla-staging/job/valerii/job/vp-longevity-10gb-3h-test/56
In short: works. Ready for review and merge.

**Comment by fruch** [comment]
It was tested on Scylla from 2023.1 ?

I.e. we are still waiting for a backport for it to work ?

**Review by fruch — APPROVED** [review]
LGTM

**Comment by vponomaryov** [comment]
> It was tested on Scylla from 2023.1 ?

It was tested on latest dev enterprise.

> I.e. we are still waiting for a backport for it to work ?

If you mean `2023.1` then yes.

**Pull request merged**